src/utils.ts

```diff
@@ -16,7 +16,7 @@ export const checkDependencies = (
   dependencies: Record<string, string>,
 ): Packages | typeof IMPORT_KIND_PLAYWRIGHT | null => {
   const packages: Packages = {}
-  if (!dependencies) return null
+  if (!dependencies || Object.keys(dependencies).length === 0) return null
```

Comment:
I think we should check on the `packages` variable instead of the dependencies variable. Since in my case, both are filled but only `devDependencies` does contain the playwright related packages.

Reply:
@mxschmitt yeah! That's true, but it will cover case when `dependencies` or `devDependencies` are defined, but empty. It should return `null`, and then we can check if they have some of **playwright** dependencies. So we check length of `packages` variable in the end. And in `readPackage` we can check result with `null`

Reply:
LGTM now 👍